app/assets/stylesheets/pages/_index.scss

```diff
@@ -23,11 +23,13 @@
   h1 {
     color: $orange;
     font-size: 5em;
+    -webkit-text-stroke: 1px black;
```

Comment:
-webkit- is a vendor prefix. It makes it work on chrome and safari, but it won't work on firefox and opera unless you add -moz- and -o-. Unless bourbon does all the vendor prefixes for you, in which case you might not even need the -webkit-